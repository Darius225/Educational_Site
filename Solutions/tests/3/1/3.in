10
8 18 15 19 4 17 15 13 17 8 
6
4
0
15
13
5
5
