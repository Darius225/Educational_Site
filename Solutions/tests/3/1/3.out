4
8
-1
-1
17
17
